src/Utils/RajaUmpireUtils.hpp

```diff
@@ -56,42 +56,18 @@
 #ifndef HIOP_RAJA_UMPIRE_UTILS
 #define HIOP_RAJA_UMPIRE_UTILS
 
-#pragma once
-
-//Umpire is always enabled with RAJA
-#if defined(HIOP_USE_RAJA)
-
 #include <string>
 #include <sstream>  
 
-#include <umpire/Allocator.hpp>
-#include <umpire/ResourceManager.hpp>
-
-#include <RAJA/RAJA.hpp>
-
 namespace hiop {
   /** 
-   *  Returns a string with information Umpire keeps about the address passed as the argument, 
-   *  or the "__not_managed_by_umpire__" if Umpire does not have a record of the address.
-   *
+   * Returns a string with memory space Umpire keeps about the address passed as the argument, 
+   * "__not_managed_by_umpire__" if Umpire does not have a record of the address, or 
+   * "__info_not_available__HiOp_not_built_with_Umpire__" if HiOp was not built with Umpire.
+   * 
+   * The function should be used for debugging purposes only. 
    */
-  std::string get_umpire_mem_address_info(void* address)
-  {
-    auto& rm = umpire::ResourceManager::getInstance();
-    if(rm.hasAllocator(address)) {
-      auto found_allocator = rm.getAllocator(address);
-      std::stringstream ss;
-      ss << "Allocated on '" << found_allocator.getName() << "'   ";
-      ss << "Platform " << static_cast<int>(found_allocator.getPlatform()) << "   "; 
-      ss << "Size [" << found_allocator.getSize(address) << "]";
-      return ss.str();
-    } else {
-      return std::string("__not_managed_by_Umpire__");
-    }
+  std::string get_umpire_mem_address_info(void* address);
```

Comment:
Where is this function called in HiOp?

Reply:
it's not, it's for debugging only, but I tested it manually.